doc/releases.md

```diff
@@ -14,30 +14,33 @@ CDI adheres to the [semantic version definitions](https://semver.org/) format of
 
 ### Releasing a New Version
 
- The version number is tracked in several files in CDI as well as through a git tag.  To reduce the chance of human error, a help script is used to change the version in all known locations.
+Release branches are used to isolate a stable version of CDI.  Git tags are used within these release branches to track incrementation of Minor and Patch versions.  When a Major incrementation occurs, a new stable branch should be created corresponding to the release.
 
-     DO NOT EDIT ANY VERSION STRINGS IN CDI!!
+- Release branches should adhere to the `release-v#.#.#` pattern.
 
-1. Set the new release version
+- Tags should adhere to the `v#.#.#(-alpha.#)*` pattern.
```

Comment:
why parenthesis here? If you mean optional then I think square brackets are more typical, eg:
`v#.#.#[-alpha.#*`
Also, confused by the `*`. Is that a regex to mean _repeated 0 or more times_?